x-pack/plugins/enterprise_search/server/routes/app_search/reference_application.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import { RouteDependencies } from '../../plugin';
+
+export function registerReferenceApplicationRoutes({
+  router,
+  enterpriseSearchRequestHandler,
+}: RouteDependencies) {
+  router.get(
+    {
+      path: '/api/app_search/engines/{engineName}/reference_application/field_config',
```

Comment:
[not blocking, just wondering] should we rename the server route/file to `/search_ui` as well? 🤷

Reply:
I don't know. I've been trying to keep these 1 to 1 with the actual ent-search API, but I know we already have a bunch of API endpoints that are not 1 to 1, so I'm not opposed to changing it.

Reply:
37dcad43c7e0fc1b97be33b8f8396bcdfc9dab83